src/go/monitor/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// serviceMetrics holds metrics for a service
+type serviceMetrics struct {
+	VisitTotal prometheus.Counter
+}
+
+// newServiceMetrics registers metrics
+func newServiceMetrics() serviceMetrics {
+	return serviceMetrics{
+		VisitTotal: promauto.NewCounter(prometheus.CounterOpts{
+			Name: "myapp_visit_total",
```

Comment:
Can we extract `myapp` into a top level constant and add a param to this `newServiceMetrics` top be used as prefix to all the metrics?